lms/djangoapps/django_comment_client/forum/views.py

```diff
@@ -402,7 +404,9 @@ def single_thread(request, course_key, discussion_id, thread_id):
             'user_cohort': user_cohort,
             'sort_preference': cc_user.default_sort_key,
             'category_map': course_settings["category_map"],
-            'course_settings': json.dumps(course_settings)
+            'course_settings': json.dumps(course_settings),
+            'disable_courseware_js': True,
+            'uses_pattern_library': True,
```

Comment:
I added a section to the "How to add a new page" wiki page a couple of weeks ago:

https://openedx.atlassian.net/wiki/display/FEDX/How+to+add+a+new+page+to+LMS+or+Studio#HowtoaddanewpagetoLMSorStudio-UsethePatternLibrary

Let me know if you think it needs more detail.